### PR TITLE
[TACHYON-368] Tests for MASTER_HOSTNAME_LISTENING config option

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

File: integration-tests/src/test/java/tachyon/master/MasterClientTest.java [hunk 1]
```diff
@@ -33,10 +33,10 @@
 import tachyon.thrift.NoWorkerException;
 
 /**
- * Though its name indicates that it provide the unit tests for {@link tachyon.master.MasterClient},
+ * Though its name indicates that it provides the tests for {@link tachyon.master.MasterClient},
```
Comment (calvinjia) [inline integration-tests/src/test/java/tachyon/master/MasterClientTest.java]: Are the only changes to this file clean up?

**Comment by calvinjia** [comment]
+1 LGTM

File: integration-tests/src/test/java/tachyon/master/MasterClientTest.java (hunk 1 above)
Comment (gpang) [inline integration-tests/src/test/java/tachyon/master/MasterClientTest.java]: Yes, only cleanup changes were done to this file.

**Comment by apc999** [comment]
I know this is late but still want to put a comment for future reference:

For tests that are located in integration tests (typically bringing up a local cluster), we name them like FooBarIntegrationTest. In this case, it will be MasterHostnameListeningIntegrationTest.java

I will make another PR to rename tests files that are not consistent to this pattern yet (there are more than one in addition to this new test).

**Pull request merged**